src/pretix/base/settings.py

```diff
@@ -2002,6 +2002,24 @@ def unserialize(cls, s):
 You can view the payment information and the status of your order at
 {url}
 
+Best regards,
+Your {event} team"""))
+    },
+    'mail_subject_order_incomplete_payment': {
+        'type': LazyI18nString,
+        'default': LazyI18nString.from_gettext(gettext_noop("Your order received an incomplete payment: {code}")),
```

Comment:
I'll suggest a shorter version:

```suggestion
        'default': LazyI18nString.from_gettext(gettext_noop("Incomplete payment received: {code}")),
```